src/autocomplete/shared/parse-query-with-cursor.ts

```diff
@@ -33,6 +34,10 @@ export function parseYQLQueryWithCursor(queryWithCursor: string): YQLAutocomplet
     return parseYQLQuery(...separateQueryAndCursor(queryWithCursor));
 }
 
+export function parseYQQueryWithCursor(queryWithCursor: string): YQLAutocompleteResult {
+    return parseYQQuery(...separateQueryAndCursor(queryWithCursor));
```

Comment:
We don't have abbreviations in all caps in our project. So please in the next PR, refactor your functions to `parseYqQueryWithCursor`, `parseYqlQueryWithCursor`.